image/spec/specs-go/v1/image_deprecated.go

```diff
@@ -0,0 +1,31 @@
+// Package v1 is deprecated and moved to github.com/moby/docker-image-spec/specs-go/v1
+//
+// Deprecated: use github.com/moby/docker-image-spec/specs-go instead.
+package v1
+
+import v1 "github.com/moby/docker-image-spec/specs-go/v1"
+
+// DockerOCIImageMediaType is the media-type used for Docker Image spec images.
+//
+// Deprecated: use [dockerspec.DockerOCIImageMediaType].
```

Comment:
```suggestion
// Deprecated: use [v1.DockerOCIImageMediaType].
```

Reply:
🤦 I forgot I disabled "in comments" for refactor/rename in my IDE, as it was often too greedy. Should be fixed now.